### ISSUE TITLE
Bug: Kernel panic on memory plug

### ISSUE THREAD

**Issue by vadim2404** [neondatabase/autoscaling#450]
## Other logs, links

- ...

**Comment by vadim2404** [comment]
@knizhnik created a branch with in-memory cache neondatabase/neon#4891

**Comment by vadim2404** [comment]
Should we merge mmap version of the cache and remove our tmpfs hacks?

**Comment by sharnoff** [comment]
Yes, once reviewed & passing tests, etc. I'm not familiar enough with C to review it.

**Comment by sharnoff** [comment]
This has been mitigated by moving the file cache to disk, but the underlying kernel issue is still present, so: leaving this issue open, but reducing priority.

**Comment by sharnoff** [comment]
We recently updated the kernel (#638), and should check whether this issue is now fixed because of it.
